x-pack/plugins/synthetics/server/alert_rules/status_rule/monitor_status_rule.ts

```diff
@@ -63,19 +66,21 @@ export const registerSyntheticsStatusCheckRule = (
     isExportable: true,
     minimumLicenseRequired: 'basic',
     doesSetRecoveryContext: true,
-    async executor({ state, params, services, startedAt, spaceId, previousStartedAt }) {
+    async executor({ state, params, services, spaceId, previousStartedAt }) {
       const ruleState = state as SyntheticsCommonState;
 
       const { basePath } = server;
       const {
         alertFactory,
         getAlertUuid,
-        getAlertStartedDate,
         savedObjectsClient,
         scopedClusterClient,
         alertWithLifecycle,
+        uiSettingsClient,
       } = services;
 
+      const dateFormat = await uiSettingsClient.get('dateFormat');
```

Comment:
@shahzad31 @justinkambic Not sure if I need to go ahead and request the tz from the ui settings as well.